lib/vmdb-logger.rb

```diff
@@ -0,0 +1,164 @@
+require 'logger'
+
+class VMDBLogger < Logger
+  def initialize(*args)
+    super
+    self.level = INFO
+
+    # HACK: ActiveSupport monkey patches the standard Ruby Logger#initialize
+    # method to set @formatter to a SimpleFormatter.
+    #
+    # The ActiveSupport Logger patches are deprecated in Rails 3.1.1 in favor of
+    # ActiveSupport::BufferedLogger, so this hack may not be needed in future
+    # version of Rails.
+    self.formatter = Formatter.new
```

Comment:
LOL this comment.  It might be worthwhile see if that hack is still needed in a followup.